### PR TITLE
Custom filename for reports

### LINKED ISSUE #225: Renaming report file
Is there a way to rename the report file on generation time?

### PR DESCRIPTION
**Pull request by Poeschl**
This PR holds the option to specify the name of the generated report file.
It doesn't use gradle's build-in report features but for now does the job.

Its closes #225 und was inspired by #48 

I thought about a way to test the `reportfileName` option, but without success. If you have something in your mind about it or any remarks on the changes feel free to write.

### CODE REVIEW

File: src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy [hunk 1]
```diff
@@ -19,13 +19,7 @@ import com.github.benmanes.gradle.versions.reporter.JsonReporter
 import com.github.benmanes.gradle.versions.reporter.PlainTextReporter
 import com.github.benmanes.gradle.versions.reporter.Reporter
 import com.github.benmanes.gradle.versions.reporter.XmlReporter
-import com.github.benmanes.gradle.versions.reporter.result.DependenciesGroup
```
Comment (vanniktech) [inline src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy]: I don't know what the convention is here but I personally don't like star imports and since it has been changed I'd advise to revert this change.

File: src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesTask.groovy [hunk 2]
```diff
@@ -67,4 +70,7 @@ class DependencyUpdatesTask extends DefaultTask {
 
   /** Returns the outputDir destination. */
   String outputDirectory() { System.properties['outputDir'] ?: outputDir }
+
+  /** Returns the useProjectAsFilename boolean. */
```
Comment (vanniktech) [inline src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesTask.groovy]: Documentation is wrong here.

File: src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy (hunk 1 above)
Comment (Poeschl) [reply src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy]: It seems that the company convention kicked in, will be reverted.

**Comment by Poeschl** [comment]
Review integrated

**Comment by vanniktech** [comment]
Thanks.
